Add dockerfile

AntoniaChroni opened the pull request:
The following changes were made in the main directory:

- Updated README.md
- Added Dockerfile
- Added scripts folder
    ├── install_bioc.r
    └── install_github.r

Please review file content in the folders and test if the Docker image runs.

Commit:
Added files related to Docker

- Created Dockerfile and docker image
- Updated README.md with instructions on how to reproduce the code in this repository
- Added scripts folder with install_bioc.r and install_github.r scripts

Commit:
Update README.md

jharenza commented on Dockerfile:
```diff
@@ -0,0 +1,87 @@
+FROM rocker/tidyverse:4.2
+MAINTAINER rokita@chop.edu
```
```suggestion
MAINTAINER rokita@chop.edu
```

Since this is going to be stored in your CAVATICA registry, you could put yourself here. Though it doesn't matter per se for this. Please add me as an admin for your docker image, so that I can make changes as well and/or add others to contribute. Thanks!

jharenza commented on Dockerfile:
```diff
@@ -0,0 +1,87 @@
+  "requests==2.27.1" \
+  "urllib3==1.26.8"
+
+# Install oncokb
+RUN git clone https://github.com/oncokb/oncokb-annotator.git /home/oncokb-annotator
```
you probably do not need this for now, and it needs an auth token to work.

jharenza commented on Dockerfile:
```diff
@@ -0,0 +1,87 @@
+RUN apt-get -y --no-install-recommends install \
+    cmake
+
+# install R packages from CRAN
+RUN install2.r \
+	BiocManager \
+  ComplexHeatmap \
+  data.table \
+  ggpubr \
+  ggthemes \
+  msigdbr \
+	optparse \
+	pheatmap \
+	RColorBrewer \
+	survival \
+  survMisc \
+  survminer \
+  tidytext
```
These are a good start but I think you also may need others, as you had indicated in your PR: #1 
```
library(flextable)
```

Also, if you think of anything else you might need off the bat which you tend to use or we may need, ie - vroom for loading the large MAF files, feel free to add here and rebuild.

jharenza reviewed (changes requested):
looks good but made a few suggestions

Commit:
Updated Dockerfile

- Changed Maintainer
- Added flextable, vroom R packages
- Removed unnecessary code at the end for onco

jharenza reviewed (approved):
builds successfully and runs

jharenza reviewed (commented):
Actually, can you merge main into this branch and remove the DS store files? Then we can merge

jharenza reviewed (commented):
Actually, can you merge main into this branch and remove the DS store files? Then we can merge

Commit:
removed .DS_Store

jharenza committed:
remove DS store files

jharenza committed:
remove a few more

jharenza committed:
Merge branch 'main' into add-files
